src/Gui/MainWindow.cpp

```diff
@@ -2250,3 +2293,4 @@ ActionStyleEvent::Style ActionStyleEvent::getType() const
 
 
 #include "moc_MainWindow.cpp"
+#include "MainWindow.moc"
```

Comment:
What is that? How is it related to this pr?

Reply:
`DimensionWidget` uses the `Q_OBJECT` macro so Qt MOC will automatically generate (and need) this file so it's included here.

Reply:
Thanks for the explanation